api/controller/datasets.py

```diff
@@ -250,9 +253,10 @@ def upload_data(
 
     """
     try:
+        subject_id = parse_token(request.cookies.get(RAPID_ACCESS_TOKEN)).subject
```

Comment:
Should we use _get_subject_id instead?

Reply:
_get_subject_id returns a string if the token doesn't exist and I don't think having this string is correct here?